packages/aws-cdk/lib/api/cxapp/stacks.ts

```diff
@@ -225,6 +226,22 @@ export class AppStacks {
     }
   }
 
+  public getTagsFromStackMetadata(stack: SelectedStack): Tags {
+    const tags = [];
+    for (const id of Object.keys(stack.metadata)) {
+      const metadata = stack.metadata[id];
+      for (const entry of metadata) {
+        switch (entry.type) {
```

Comment:
What's wrong with an "if"?

Reply:
Me bad. I will change this today when I got sometime. (many meetings). Thanks for the review.